### ISSUE TITLE
inspect_response() + view(response)  opens TextEditor rather than Browser

### ISSUE THREAD

**Issue by laoyur** [scrapy-plugins/scrapy-splash#82]
```
    def parse(self, response):
        """
        :param response:
        :type response: SplashTextResponse
        :return:
        :rtype:
        """
        inspect_response(response, self) 
```

inspection terminal:

```
[s] Available Scrapy objects:
[s]   crawler    <scrapy.crawler.Crawler object at 0x112c7ecd0>
[s]   item       {}
[s]   request    <GET http://s.weibo.com/weibo/funny?topnav=1&wvr=6&b=1 via http://192.168.99.100:8050/render.html>
[s]   response   <200 http://s.weibo.com/weibo/funny?topnav=1&wvr=6&b=1>
[s]   settings   <scrapy.settings.Settings object at 0x112bf6610>
[s] Useful shortcuts:
[s]   shelp()           Shell help (print this help)
[s]   view(response)    View response in a browser
In [1]: view(response)    // <---- opens TextEdit
Out[1]: 
True
```

Question: how to force open browser with view(response)  ?

**Comment by FreeTymeKiyan** [comment]
I have the same issue. No idea why TextEdit is opened not browser.

**Comment by FreeTymeKiyan** [comment]
Just noticed that the response is saved as a temporary .txt file, which explains why TextEdit is opened by default. 

The problem now becomes why it's a text file not an HTML.

**Comment by FreeTymeKiyan** [comment]
Alright, I got it. Basically, the responses we got from Splash can be:

- SplashResponse is returned for binary Splash responses - e.g. for /render.png responses;
- SplashTextResponse is returned when the result is text - e.g. for /render.html responses;
- SplashJsonResponse is returned when the result is a JSON object - e.g. for /render.json responses or /execute responses when script returns a Lua table.

If it's `SplashTextResponse` then it's text file. 

I set `dont_process_response=True` when building the request. Now `view(response)` opens it in browser. 🙌
